src/test/java/core/basesyntax/service/impl/WriteDataToStorageImplTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.TransactionStorage;
+import core.basesyntax.service.file.WriteDataToStorageService;
+import org.junit.After;
+import org.junit.Test;
+
+public class WriteDataToStorageImplTest {
+    private WriteDataToStorageService writeDataToStorageService = new WriteDataToStorageImpl();
+    private TransactionStorage transactionStorage = new TransactionStorage();
+
+    @Test
+    public void writeDataToStorage_firstLine_Ok() {
```

Comment:
First part of test method name should be name of the method you test 
```suggestion
    public void writeData_firstLine_Ok() {
```